Automattic/wp-calypso
# Improves the page load speed for the Blaze app

I am noticing a delay in the Blaze Dashboard app's response when it is loading. The main problem is that you can see the dashboard already rendered with all the posts, but you can't change tabs until the page fully loads. 

The async functions that we have in Calypso + Blaze are preventing the app from fully loading quickly, and until the `load` event is triggered in the browser, the `page` package won't take any pop state from the hash.

From the user's point of view, you will see the tab changing, but the actual tab won't be rendered. After some time, it will start working.

![2024-04-23 16 10 12](https://github.com/Automattic/wp-calypso/assets/1258162/a3c3af18-4c3f-415d-8ff5-2ad6e59cfc38)

## Proposed Changes

* Show the first dashboard page when the `load` event is triggered.

## Testing Instructions

* Connect to your sandbox and execute `bin/install-plugin.sh blaze fix/blaze-app-load-speed`
* Move the traffic from `widgets.wp.com` to your sandbox
* Open any self-hosted site (or any other site using Jetpack Blaze, like Pressable)
* Go to Tools->Advertising and verify that the page loads quickly and that you can change the tabs without seeing the previous behavior

## Pre-merge Checklist
